execution/executor-benchmark/src/main.rs

```diff
@@ -193,7 +193,7 @@ struct Opt {
     hotspot_probability: Option<f32>,
 
     #[clap(long)]
-    concurrency_level: Option<usize>,
+    execution_threads: Option<usize>,
```

Comment:
This rename can break many informal docs🤔

Reply:
explanation of why I am changing this: 

for parallelism, I think that "--concurrency-level 1 --num-executor-shards 48" is a bit confusing overloading of the flags

we should have one flag of number of cores given (allowed to be used) by the execution stage. and then additional flags if sharding needs it.

currently --concurrency-level is that flag, so I would assume 8 means 8 cores are given
and then having --num-executor-shards be 8, if you want to run 8 BlockSTMs.

I can leave concurrency-level, but that is a bit ambiguous, so changing to execution_threads (in the table we print "exe 8" for example already), because that is what it means


I prefer breaking a few old commands for clarity. I can add some comment though